CmswebApI/DTOs/CourseDto.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace CmswebApI.DTOs
+{
+    public class CourseDto
+    {
+        public int CourseID { get; set; }
+        public string CourseName { get; set; }
+        public int CourseDuration { get; set; }
+        public string Source { get; } = "I am from Dto";
+        
+         [JsonConverter(typeof(JsonStringEnumConverter))]
```

Comment:
Why have you done this?

Reply:
Because, it was expecting and returning an integer value for Enum by default.